src/main/java/com/hypherionmc/modpublisher/plugin/ModPublisherGradleExtension.java

```diff
@@ -501,7 +501,6 @@ public void changelog(String changelog) {
      * Options related to publishing GitHub Releases
      */
     @Getter
-    @Setter
```

Comment:
import should be removed too

Reply:
Good catch. Kotlin REALLY doesn't like the setters generated by lombok, which is why everything else has handwritten setters... Hate it tbh.

Well technically they work, but then the groovy syntax is 

tag = "tag"

and kotlin is

setTag("tag")

Reply:
> Well technically they work, but then the groovy syntax is
> 
> `tag = "tag"`
> 
> and kotlin is
> 
> `setTag("tag")`

Tbh, I'd be happy with that, but I guess this ought to be consistent with the rest of the codebase.